nxdl.xsd

```diff
@@ -871,6 +871,11 @@
 							* ``scaler`` = 0-D data to be plotted
 							* ``spectrum`` = 1-D data to be plotted
 							* ``image`` = 2-D data to be plotted
+							* ``rgbimage`` = 2-D images with colour as added 3rd dimension
```

Comment:
Can these match https://github.com/nexusformat/definitions/blame/master/manual/source/design.rst#L186

Reply:
Those names in the design chapter were added 6 years ago.  I'll update this PR to use the hyphenated versions AND the associated text.

Reply:
with recent changes, they match now